rundeck/job.go

```diff
@@ -198,6 +199,31 @@ type JobDispatch struct {
 	RankOrder       string `xml:"rankOrder,omitempty"`
 }
 
+type JobSchedule struct {
+	Month           jobScheduleMonth    `xml:"month"`
+	Year            jobScheduleYear     `xml:"year"`
+	Weekday         jobScheduleWeekDay  `xml:"weekday"`
+	Time            jobScheduleTime     `xml:"time"`
+}
+
+type jobScheduleMonth struct {
+	Month string `xml:"month,attr"`
+}
+
+type jobScheduleYear struct {
+	Year string `xml:"year,attr"`
+}
+
+type jobScheduleWeekDay struct {
+	Weekday string `xml:"weekday,attr"`
```

Comment:
When i export a job with schedule i get the following:
```xml
<joblist>
    ...
    <schedule>
      <month month='*' />
      <time hour='01' minute='00' seconds='0' />
      <weekday day='*' />
      <year year='*' />
    </schedule>
    ...
</joblist>
```
That means your attribute for WeekDay is not correct.
```go
type jobScheduleWeekDay struct {
	XMLName xml.Name `xml:"weekday"`
	Weekday string   `xml:"day,attr"`
}
```